### ISSUE TITLE
Run standalone without installing

### ISSUE THREAD

**Issue by KE6MTO** [openrocket/openrocket#2376]
I've created an ARC team and our plan is to use OpenRocket as our primary CAD software. The issue we run into is the school removes any software that gets installed at shutdown. What would be great is if OpenRocket can be run off an external hard drive or USB drive. That way no install is needed and makes it super easy to use on school computers. Something to consider.

**Comment by JoePfeiffer** [comment]
This already works -- I just downloaded a copy of 22.02, installed it to a USB drive, and ran it from the drive.  I had to use the file browser to navigate to the OpenRocket directory on the drive and then run OpenRocket.exe, but it worked. I created a .ork file, and saved it to the USB drive too.

There may some issues due to your environment, of course -- I expect desktop shortcuts get deleted upon shutdown, and I can't know whether your preferences are preserved. But OR itself is working as you requested.
